### ISSUE TITLE
Initializing SweetAlert does resize my form

### ISSUE THREAD

**Issue by Laiteux** [Jungwoo-An/SweetAlertSharp#4]
Hello,

When initializing a `SweetAlert` with `new SweetAlert();`, this is what happens:
![](https://laiteux.dev/gkdnzh.gif)

As you can see, my form is getting resized, in a smaller way.
Why that?

This is my button code:
```cs
private void button1_Click(object sender, EventArgs e)
{
    new SweetAlert();
}
```

I am using a Windows.Forms project on .NET Framework 4.7.2.

Regards,
Matt.

**Comment by Laiteux** [comment]
Thanks for reacting to it but what now? Haha

**Comment by Jungwoo-An** [comment]
@Laiteux Hi! thanks for your interest! ❤️

I'm sorry for the late reply. I checking this issue.

I hope to tell good news to you.

Thanks!

**Comment by Jungwoo-An** [comment]
@Laiteux I checked your code, But i can't reproduce.

Can you give me detailed example code, If you don't mind?

**Comment by Laiteux** [comment]
@Jungwoo-An Hello, here you go:

[Sandbox.Forms.zip](https://github.com/Jungwoo-An/SweetAlertSharp/files/4181531/Sandbox.Forms.zip)

**Comment by Jungwoo-An** [comment]
@Laiteux Thanks! I'll check it asap. 👍

**Comment by Laiteux** [comment]
I did some tests, and figured out this only happen on my computer.
I tried using a Windows Server 2016 Standard computer and the form didn't get resized.
It also happens when using CommonOpenFileDialog...
I have no idea where does this come from, but it isn't your library.
Sorry and thanks for your time.

I'm closing this issue, if you ever have an idea about why is this happening (because I really don't know atm), feel free to comment!

Regards,
Matt.
